units/CMakeLists.txt

```diff
@@ -22,8 +24,11 @@ if(UNITS_HEADER_ONLY)
 else(UNITS_HEADER_ONLY)
     if(UNITS_BUILD_STATIC_LIBRARY)
         add_library(units-static STATIC ${units_source_files} ${units_header_files})
+        generate_export_header(units-static BASE_NAME units)
```

Comment:
Can we leave this out of the static library build?  there doesn't seem to be a need to generate the export header for the static library.